nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -183,25 +164,27 @@
 
     protected OperationType operationType;
     protected SessionConfiguration.FlushMode flushMode;
+
     protected int batchSize = 100;
     protected int ffbatch   = 1;
 
-    protected KuduClient kuduClient;
-    private volatile KerberosUser kerberosUser;
+    protected KuduTable kuduTable;
 
     @Override
     protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
         final List<PropertyDescriptor> properties = new ArrayList<>();
         properties.add(KUDU_MASTERS);
-        properties.add(TABLE_NAME);
```

Comment:
Any reason to change the order of the properties? just curious

Reply:
updated old order